_episodes/02-data-quality.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Data Formats and Data Quality"
+teaching: 0
+exercises: 0
+questions:
+- How do I read and write tabular data?
```

Comment:
I think everything in the questions, objectives, and keypoints sections need to be in quotes.